### DISCUSSION TITLE
Stores and indexed accessors

### DISCUSSION THREAD

**Discussion by demoran23** [solidjs/solid#1440] [Q&A] [answered]
I have a store called `downloads` in the shape of

```typescript
{ [key: string]: boolean }
```

and I want to use it in one of my components.

When trying to use it in my component, the store updates do not trigger a re-render.

I am accessing the store property dynamically, like

```typescript
const downloaded = createMemo(() => downloads[props.link]);
```

And I'm unsure how to make it work.

**Comment by atk** [comment, marked as answer]
You don't need a memo to access the store. How do you update the store?

**Comment by demoran23** [comment]
Hmm, if this syntax isn't an issue, then it's likely a difference in runtime environment between my Chrome extension's background script and the UI.  I'll try approaching the problem in that direction.

**Reply by demoran23** [reply]
Okay, that did the trick.  This was a "you can't talk across these boundaries" issue, rather than a "you are running into problems with solid syntax" issue.

Thanks, @atk!
